xcodeproj/internal/output_files.bzl

```diff
@@ -0,0 +1,158 @@
+"""Module containing functions dealing with target output files."""
+
+load(":output_group_map.bzl", "output_group_map")
+
+# Utility
+
+def _create(*, direct_outputs = None, attrs_info = None, transitive_infos):
```

Comment:
Not sure if the signature for this has solidified yet. If they are, we may want to add doc comments.